spec/models/rent_spec.rb

```diff
@@ -0,0 +1,13 @@
+require 'rails_helper'
+
+describe Rent do
+  subject(:rent) { create(:rent) }
+
+  it { is_expected.to validate_presence_of :user_id }
+  it { is_expected.to validate_presence_of :book_id }
+  it { is_expected.to validate_presence_of :from }
+  it { is_expected.to validate_presence_of :to }
+  it { is_expected.to satisfy('from cannot be greater than to') { |rent| rent.from <= rent.to } }
```

Comment:
🤔  I've never used `satisfy`, I'll have to look it up.
We usually do something like `it { expect(rent.from).to be <= rent.to }`

Reply:
Oks, I will try do it like that.